presto-docs/src/main/sphinx/release/release-0.261.rst

```diff
@@ -0,0 +1,45 @@
+=============
+Release 0.261
+=============
+
+**Highlights**
+==============
```

Comment:
you can remove the highlights section if there is nothing worth highlighting .

Reply:
I added some. I didn't realize that was something I needed to populate.